src/utils/formatters.js

```diff
@@ -149,8 +149,8 @@ export const formatHoustonTime = possibleTimeObject => {
   try {
     const houstonTime = format(
       possibleTimeObject,
-      "yyyy-MM-dd'T'hh:mm:ssxxx", // ISO 8601 without "Z" timezones.
-    );
+      "yyyy-MM-dd'T'HH:mm:ssxxx",
```

Comment:
Hm. Does this fix JH's 24-hr-clock ticket?

Reply:
It does! `HH` is 24 hour time, `hh` is 12 hour time (expecting an AM or PM).